chpl/chpl-resources/src/main/resources/email.properties

```
@@ -5,6 +5,8 @@
 # * key/value pair in [Tomcat]/conf/email-override.properties file.
 #########################################################################################
 
+#URL to ONC website used in default HTML footer
+footer.url=https://inquiry.healthit.gov/support/servicedesk/customer/user/login?destination=site%2Fforum
```

I went down a rabbit hole on this one

```suggestion
footer.url=https://inquiry.healthit.gov/support/servicedesk/customer/user/login?destination=site/forum
```

In the RFC for URIs/URLs the query section allows `/` and `?` as valid characters and suggests that it's better to not URL encode those characters: https://tools.ietf.org/html/rfc3986#section-3.4 Also, this URL does actually work :)